jupyterlab_bigquery/jupyterlab_bigquery/handlers.py

```diff
@@ -118,6 +118,14 @@ def get_table_details(client, table_id):
       }
   }
 
+def get_table_preview(client, table_id):
+  table = client.get_table(table_id)
+  rows = client.list_rows(table, max_results=100)
+  return {
+    'fields': [field.name for field in rows.schema],
+    'rows': [[value.__str__() if isinstance(value, datetime.date) else value for value in row.values()] for row in rows]
```

Comment:
For now, this handles date objects along with strings, but I'm not sure what other objects might come through from BQ table entries that I need to handle.

Reply:
update: I will make changes to handle all possible schema types and add it as a commit to this pr